src/commons/api/ca.rs

```diff
@@ -1836,22 +1404,6 @@ impl fmt::Display for RepoStatus {
                     "Next contact on or before: {}",
                     self.next_exchange_before().to_rfc3339()
                 )?;
```

Comment:
This information is currently shown by `krillc repo status`. Is it intentional that it is no longer output here (nor can I see any changes to `krillc` to fetch the information a different way), or is this on the TODO list?

Reply:
not sure what you mean, this is being shown.

Reply:
I’ll check again, it wasn’t showing when I checked before.